### ISSUE TITLE
Domain Not allowed

### ISSUE THREAD

**Issue by alifopensrc** [RainLoop/rainloop-webmail#919]
Hi all,

Latest fresh install on debian 8 x64 renders the "Domain not allowed" error. Roundcube works. So that eliminates any possibility that mailserver not configured properly.

I have checked for guides from 3 issues namely #296, #720 & #15 but seems futile.
Please advice.

Thank you.
-AOS.

**Comment by alifopensrc** [comment]
Hi,

I've checked on a 2nd Debian fresh install and the problem persist. Could there be a bug in the latest distrib? 

Please help verify this.

Thank you.
-AOS.

**Comment by RainLoop** [comment]
Hi @alifopensrc,

Have you configured your domain in "Admin Panel / Domains" section?

**Comment by alifopensrc** [comment]
Yes i have done all that. As i mentioned i have visited all three posts regarding this and try every single combination of settings for it. Can anyone or the authors try to set up using the latest version available online to test it out and verify if its just me. Because at this point i have tried 3 different machines already.

Thank you.
-AOS.

**Comment by RainLoop** [comment]
Could you send me (support@rainloop.net) a configuration screenshot?

for example:

![image](https://cloud.githubusercontent.com/assets/5530092/12125863/db517b2c-b3fe-11e5-89b5-dc94e027b841.png)

**Comment by alleste** [comment]
I have the same problem: I have configured the configured the domain at the admin page and the "Test" button does work. I inserted the domain xxx.xxx-xxx.de in the whitelist. When I try to log in to the mail account I get the error message "domain is not allowed".

**Comment by CrashSYS** [comment]
I have the same problem, 

Under Login - "Login Screen"
     [ domain-1.com   ]


Under Domain - Server is added, a user to the white list, with full email address 
> john@domain-1.com <  and  also just > john < is added.

And still I keep getting Domain is Not Allowed.

I tried a Secure way and a Non-Secure Way,  I have a Linux Server, Apache, Cpanel

Please reply, I really liked the demo, and want to tried out before purchasing.

**Comment by bigkiandi-projects** [comment]
## domain is now allowed

this is working solution that you should try. Please take a look in image below : 

<img src="https://goldentranswisata.com/strorage/rl.png">

So, First step is that you must fill name input as you email domain after @ character. example if your email domain is user@mydomain.com, you should typing @mydomain.com. SEE ON THE RED INFORMATION ON THE RIGHT SIDE.

**Comment by gitterdoc** [comment]
It's possible to say "all domains try to use default SMTP/IMAP Configuration"?
I don't want to add for each domain an alias.

**Comment by Matthiasuser** [comment]
I have the same problem. Unfortunately I can not see the image posted of bigkiandi-projects. I added @mydomain.de in the whitelist but it still does not work.

Can anyone help, please!

**Comment by Salmon-Bard** [comment]
The trick is when adding a domain to allowed domain list you Must make sure that the value entered in the top first filed "the name filed"  in the adding form is a qualified domain name in the form "domain.extension" DO NOT use any other name than the domain used in the address that will use this allowed domain. So this means that all your emails accessing this account in the form "email@doamin.extensio"
